client/Hooks/useRouteConfig/useRouteConfig.spec.ts

```diff
@@ -0,0 +1,419 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { useRouteConfig } from 'Hooks';
+import { PageConfig, RouterConfig } from 'Hooks/types';
+import HomePageComponent from '../../../test_common/resources/Home.view';
```

Comment:
Not an `useRouteConfig.assets.tsx` file? While I see the intent of sharing a mock X page, I rather these be kept with the hook in this case

Reply:
I agree with moving these into a single file would be nice. Not so sure about keeping assets next to it though

Reply:
When I reviewed, I saw these as test fixtures. We have prevision for such things ins a `*.assets.ts` file - assets is a catch all for constants and things re used across test/storybook. I would class these stubs as such.

Reply:
Would you expect both types and test stubs to both be part of an `*.assets.ts` file? That seems like something that shouldn't be mixed

Reply:
Agree with asset (given purpose of this) - or a generic generator that can create components in this test (and maybe others - see below comment)